mantidproject/mantid
# Script window conflicting with console launch

Fixes #13417

Scripting window now checks to see if the command line argument (if it is a script) matches a previously used argument to avoid opening duplicate scripts with calls from the command line.
# To Test
- Run Mantid from the command line with the name of the script as an argument 
- Ensure the script opens automatically in a scripting window
- Close Mantid
- Run Mantid again from the command line with the same script as an argument
- Ensure that this does not open a new copy of the script and only one copy is open in the scripting window
